3-platform/warehouse/db_postgres/src/main/resources/db/changelog/changes/03-create-inventories-table.sql

```diff
@@ -0,0 +1,24 @@
+-- inventories table
+
```

Comment:
move sql scripts to the warehouses/db/postgres/

we are not going to store it in resources